tests/testUtils.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
I would add this to `tests/utils/wrap` instead, so eventually we can have multiple test util files.

Reply:
I used the same structure as the non-test utils in  `lib/utils.js`, should they not be the same?

Reply:
We can split into multiple utility files later if needed.

Reply:
Will improve util organization later then.